contracts/src/spaces/facets/owner/SpaceOwnerBase.sol

```diff
@@ -110,4 +110,8 @@ abstract contract SpaceOwnerBase is ISpaceOwnerBase {
         longDescription: metadata.longDescription
       });
   }
+
+  function _getTokenId(address space) internal view returns (uint256) {
```

Comment:
could this get out of sync?

Reply:
It returns the same thing as `_getSpace(space).tokenId` without extra memory.